templates/4a_compensate_image_data_round2.ipynb

```diff
@@ -151,7 +151,8 @@
     "\n",
```

Comment:
<p>just to keep the examples consistent = final_output_channel_names = ['Ecadherin', 'FAP']</p><p> </p><p>Everything else looks great!</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/angelolab/toffy/pull/366/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='27'/>